pkg/rpc/client/rpc.go

```diff
@@ -588,3 +595,28 @@ func (c *Client) CalculateValidUntilBlock() (uint32, error) {
 	}
 	return blockCount + validatorsCount, nil
 }
+
+// AddNetworkFee adds network fee for each witness script to transaction.
+func (c *Client) AddNetworkFee(tx *transaction.Transaction, acc *wallet.Account) error {
+	size := io.GetVarSize(tx)
+	if acc.Contract != nil {
+		netFee, sizeDelta := core.CalculateNetworkFee(acc.Contract.Script)
+		tx.NetworkFee += netFee
+		size += sizeDelta
+	}
+	for _, cosigner := range tx.Cosigners {
+		contract, err := c.GetContractState(cosigner.Account)
+		if err != nil {
+			return err
+		}
+		if contract == nil {
+			continue
+		}
+		netFee, sizeDelta := core.CalculateNetworkFee(contract.Script)
+		tx.NetworkFee += netFee
+		size += sizeDelta
+	}
+	//TODO:
```

Comment:
Is there an issue for it?

Reply:
I thought that it should be a part of adding policy contract (#904). Do we need a separate issue?

Reply:
Well, it makes `AddNetworkFee()` produce incorrect fee value from what I see and can subsequently lead to failing transactions. Is there any test using this function? So at minimum we need to hardcode the value here to make correct transactions.